### ISSUE TITLE
Dashboard throws error

### ISSUE THREAD

**Issue by stsievert** [stsievert/salmon#40]
When I try to load the dashboard I get an internal server error (code 500). When I look at the Docker logs, I see this traceback starting at `Jun 08 19:54:18` UTC:

``` 
[ERROR] Exception in ASGI application
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/uvicorn/protocols/http/httptools_impl.py", line 3
    result = await app(self.scope, self.receive, self.send)
  File "/opt/conda/lib/python3.7/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in
    return await self.app(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/fastapi/applications.py", line 149, in __call__
    await super().__call__(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/applications.py", line 102, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/middleware/errors.py", line 181, in __c
    raise exc from None
  File "/opt/conda/lib/python3.7/site-packages/starlette/middleware/errors.py", line 159, in __c
    await self.app(scope, receive, _send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/middleware/base.py", line 25, in __call
    response = await self.dispatch_func(request, self.call_next)
  File "/opt/conda/lib/python3.7/site-packages/starlette_prometheus/middleware.py", line 47, in
    raise e from None
  File "/opt/conda/lib/python3.7/site-packages/starlette_prometheus/middleware.py", line 43, in
    response = await call_next(request)
  File "/opt/conda/lib/python3.7/site-packages/starlette/middleware/base.py", line 45, in call_n
    task.result()
  File "/opt/conda/lib/python3.7/site-packages/starlette/middleware/base.py", line 38, in coro
    await self.app(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc from None
  File "/opt/conda/lib/python3.7/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/opt/conda/lib/python3.7/site-packages/starlette/routing.py", line 550, in __call__
    await route.handle(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/routing.py", line 227, in handle
    await self.app(scope, receive, send)
  File "/opt/conda/lib/python3.7/site-packages/starlette/routing.py", line 41, in app
    response = await func(request)
  File "/opt/conda/lib/python3.7/site-packages/fastapi/routing.py", line 197, in app
    dependant=dependant, values=values, is_coroutine=is_coroutine
  File "/opt/conda/lib/python3.7/site-packages/fastapi/routing.py", line 148, in run_endpoint_fu
    return await dependant.call(**values)
  File "/usr/src/salmon/salmon/frontend/private.py", line 383, in get_dashboard
    activity = await plotting.activity(df, start)
  File "/usr/src/salmon/salmon/frontend/plotting.py", line 73, in activity
    bin_heights, edges = np.histogram(x, bins=bins)
  File "<__array_function__ internals>", line 6, in histogram
  File "/opt/conda/lib/python3.7/site-packages/numpy/lib/histograms.py", line 795, in histogram
    bin_edges, uniform_bins = _get_bin_edges(a, bins, range, weights)
  File "/opt/conda/lib/python3.7/site-packages/numpy/lib/histograms.py", line 425, in _get_bin_e
    '`bins` must be an integer, a string, or an array')
TypeError: `bins` must be an integer, a string, or an array
```

**Comment by stsievert** [comment]
This error does not show up in the `/logs`.

**Comment by stsievert** [comment]
This is closed by #41.
